benchmark/README.md

```diff
@@ -44,9 +44,9 @@ bash scripts/configure-environment.sh
 
 | Option | Description | Default Value |
 | --- | --- | --- |
-| types | List of databases to test | cassandra,mysql,elasticsearch|
+| types | List of databases to test. The acceptable values are elasticsearch, mysql and cassandra. | cassandra,mysql,elasticsearch|
 | outputFile | Name of the result's file | index.html |
-| scaleFactors | List of scale factors for dataset (1.0 indicates 1 GB) | 1.0 |
+| scaleFactors | List of fractional scale factors for dataset (1.0 indicates 1 GB) | 0.1 |
```

Comment:
Hmm - why the change from 1.0 to 0.1 as the default?

Reply:
Tested with 0.1 successfully till now. I thought to change the value to 1.0 after testing with it.